test/test_encryption.py

```diff
@@ -2335,6 +2335,31 @@ def test_02_success(self):
         self.client_encryption.create_data_key("aws", self.master_key)
 
 
+# https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/tests/README.rst#8-bypass-spawning-mongocryptd
+class TestBypassMongocryptWithSharedLibrary(EncryptionIntegrationTest):
+    @unittest.skipUnless(os.environ.get("TEST_CRYPT_SHARED"), "crypt_shared lib is not installed")
+    def test_via_loading_shared_library(self):
+        schemas = {"db.,coll": json_data("external", "external-schema.json")}
+        opts = AutoEncryptionOpts(
+            KMS_PROVIDERS,
+            "keyvault.datakeys",
+            schema_map=schemas,
+            mongocryptd_uri="mongodb://localhost:27021/db?serverSelectionTimeoutMS=1000",
+            mongocryptd_spawn_args=[
+                "--pidfilepath=bypass-spawning-mongocryptd.pid",
+                "--port=27021",
+            ],
+            crypt_shared_lib_required=True,
+        )
+        encrypted_client = rs_or_single_client(auto_encryption_opts=opts)
+        encrypted_client.db.coll.insert_one({"unencrypted": "test"})
```

Comment:
Can we assert that this field was actually encrypted?
```
self.assertEncrypted(self.client.db.coll.find_one({})["unencrypted"])
```

Reply:
Oh I see, my mistake was that I assumed "unencrypted" was an encrypted field. Can you change the test to use the "encrypted" field instead of "unencrypted"? Encryption will still work because this test uses the shared library.